### PR TITLE
About page implementation and Cypress Testing (Styling to come later)

### LINKED ISSUE #86: US-06: As a user, I want the system to have an easy to use simple user interface, so that i can easily navigate the system.
**Tasks:**
- [x] Design a simple and intuitive user interface (UI).
- [x] Gather stakeholder feedback for UI refinement.
- [x] Conduct usability testing with users.
- [x] Develop and implement the approved UI design.
- [x] Ensure seamless integration with system functionality.
- [x] Test the UI for easy navigation.

**Acceptance Criteria:** 
- [x] The designed UI is intuitive and aligns with established UX principles.
- [x] Stakeholders provide feedback, and necessary refinements are made.
- [ ] Usability testing confirms that users can easily navigate and use the UI.
- [ ] The implemented UI seamlessly integrates with the system's functionality.
- [ ] Test results confirm that users find the UI easy to use and navigate.

### LINKED ISSUE #93: Cypress testing: About Page.

### PR DESCRIPTION
**Pull request by Kyrus2504**
About Page:

- Added basic text content to ``About`` page
- Added a file ``3AboutPage.cy.js`` that now cypress tests About Page

### CODE REVIEW

File: cypress/e2e/3AboutPage.cy.js
Comment (gracebilliris) [inline cypress/e2e/3AboutPage.cy.js:1]: Could we maybe assert that the words `Natural Language Processing` are on the page?

File: test-results.xml
Comment (gracebilliris) [inline test-results.xml:1]: This file doesn't need to be commited to github.

File: cypress/e2e/3AboutPage.cy.js
Comment (Kyrus2504) [reply cypress/e2e/3AboutPage.cy.js:1]: Can u check again now pls?

File: test-results.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (srujankarthik) [inline test-results.xml]: @Kyrus2504 can you remove this file please, you don't need to push this on to GitHub, the xml file is for our use case only for debugging purposes.

Comment (Kyrus2504) [reply test-results.xml]: Sorry! Should be fixed now :)

**Review by srujankarthik — APPROVED** [review]
alrighty, looking good!

**Review by gracebilliris — APPROVED** [review]
Cypress testing looks good! So does the About page.

**Pull request merged**